webCommander/centralHub/views.py

```diff
@@ -80,9 +80,33 @@ def artist_details(request, spotify_artist):
     '''
     artist = get_object_or_404(SpotifyArtistInfo, spotify_artist=spotify_artist)
 
-    # no need for namedtuples, albums are already objects so support attribute access
-    # the select_related() method is used to avoid the N+1 problem, see:
-    # https://www.youtube.com/watch?v=e_8JvcP1q48
+    print(f'Amount of albums registered: {artist.spotify_artist_albums_registered}')
     albums = SpotifyAlbumTracking.objects.select_related("spotify_user").filter(spotify_user=artist)
 
+    #Is there any way to test what will actually happen on the database before doing so? :/
```

Comment:
how do you mean? 

at runtime or by adding code?

at runtime you can add a breakpoint() here to first investigate

but I guess you mean in terms of adding code here?

let's walk through this scenario (or the scenarios) together later ...